assignments/closure.js

```diff
@@ -13,9 +18,41 @@ const counter = () => {
 // newCounter(); // 1
 // newCounter(); // 2
 
+let count = 0;
+return function(){
+  count = count + 1;
+  return count;
+}
```

Comment:
I believe you meant this to be inside the `counter` function, but it is inadvertently sitting out in the global scope.